### PR TITLE
chore: #4 Updated admiral min version to 1.1.0

### LINKED ISSUE #4: Update admiral version in DESCRIPTION file
### Background Information

Currently, DESCRIPTION file records a dependency on admiral 1.1.0. But this version has a bug, and admiral 1.1.1 is due to release imminently. Once this happens, the DESCRIPTION file should be updated to point to it.

### Definition of Done

Update DESCRIPTION file with following line `admiral (>= 1.1.1)` in place of existing one.

### PR DESCRIPTION
**Pull request by AndersAskeland**
Closes #4 

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral family codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [ ] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [ ] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [ ] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [ ] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [ ] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [ ] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [ ] Address any updates needed for vignettes and/or templates
- [ ] Update `NEWS.md` under the header `# admiral<ext> (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [ ] Build site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [ ] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [ ] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [ ] Link the issue in the Development Section on the right hand side.
- [ ] Address all merge conflicts and resolve appropriately
- [ ] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -20,7 +20,7 @@ URL: https://pharmaverse.github.io/admiraltemplate,
 Depends: 
     R (>= 4.1)
 Imports:
-    admiral (>= 1.0.0),
+    admiral (>= 1.1.0),
```
Comment (manciniedoardo) [inline DESCRIPTION:23]: Actually admiral was patched after the 1.1 release (see [here](https://github.com/pharmaverse/admiral/releases)) so we should point to that.

**Comment by AndersAskeland** [comment]
@manciniedoardo Out of curiosity, I've noticed that `devtools::document()` updated the RoxygenNote version recorded in the DESCRIPTION. I can't find any information across admiral of how to handle this update. Should it be updated when prompted, or kept at the current version?

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
Thanks @AndersAskeland - please see my comment above. As this is your first PR can you also add yourself as an author in the DESCRIPTION file? See [here](https://github.com/pharmaverse/admiralophtha/blob/main/DESCRIPTION) for an example.

**Comment by manciniedoardo** [comment]
> Thanks @AndersAskeland - please see my comment above. As this is your first PR can you also add yourself as an author in the DESCRIPTION file? See [here](https://github.com/pharmaverse/admiralophtha/blob/main/DESCRIPTION) for an example.

OK to leave - I think we updated the Roxygen version on admiral core recently and it's likely just percolating through.

**Review by manciniedoardo — APPROVED** [review]
LGTM! Thanks @AndersAskeland
